cpp_exercises/day2/date/date.cpp

```diff
@@ -63,3 +64,9 @@ std::string Date::toString() const {
     stringstream << std::setfill('0') << std::setw(2) << day() << "/" << std::setfill('0') << std::setw(2) << month() << "/" << std::setfill('0') << std::setw(4) << year();
     return stringstream.str();
 }
+
+void Date::addYears(int years) {
+    m_year += years;
```

Comment:
Qui c'è siamo a rischio undefined behavior in caso di overflow!

Reply:
Fatto clamp degli anni da aggiungere da -2000 a 2000

Reply:
@lupiz-0 mi raccomando, ricordati di pushare i commit